include/vcpkg/base/messages.h

```diff
@@ -554,6 +554,16 @@ namespace vcpkg
                     (msg::spec),
                     "'HEAD' means the most recent version of source code",
                     "{spec} is already installed -- not building from HEAD");
+    DECLARE_MESSAGE(AndroidHomeDirMissingProps,
+                    (msg::env_var, msg::path),
+                    "Note: 'source.properties' is code and should not be translated.",
+                    "source.properties missing in {env_var} directory: {path}");
+
+    DECLARE_MESSAGE(AndroidTripletSupported,
+                    (),
+                    "",
+                    "export prefab requires selecting a triplet selecting a toolchain where CMAKE_SYSTEM_NAME is set "
+                    "to \"android\"");
```

Comment:
I feel like this message could be simplified.

```suggestion
                    "export prefab requires a triplet that sets CMAKE_SYSTEM_NAME to \"android\"");
```

Or maybe just: `export prefab requires an Android triplet.`.